mlfinlab/tests/test_best_stock.py

```diff
@@ -0,0 +1,46 @@
+"""
+Tests Best Stock (BestStock)
+"""
+from unittest import TestCase
+import os
+import numpy as np
+import pandas as pd
+from mlfinlab.online_portfolio_selection import BestStock
+
+
+class TestBestStock(TestCase):
```

Comment:
These tests seem too shallow. Are you sure there aren't any important tests you need to add?